cardinal/ecs/persona.go

```diff
@@ -51,11 +55,20 @@ func AuthorizePersonaAddressSystem(wCtx WorldContext) error {
 		) (AuthorizePersonaAddressResult, error) {
 			msg, tx := txData.Msg, txData.Tx
 			result := AuthorizePersonaAddressResult{Success: false}
+
+			// Check if the Persona Tag exists
 			data, ok := personaTagToAddress[tx.PersonaTag]
```

Comment:
if we want to allow both lowercase/uppercase characters for persona tag, we need to make this check case insensitive. the problem with this right now is i create a persona tag named `coolmage` and `CoolMage`, both of them will succeed creating ambiguity.

Reply:
good idea @technicallyty, I knew we needed to do this fix but impl was looking thic